tools/exporters/clinical_export.py

```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env python
+
+"""Export MedBook clinical data into rectangular files.  all fields from the form will be exported
+   Export metadata files for cbio import
+
+Usage:
+./clinical_export.py --sample_group_id (sample_group_id) --form_id (form_id)  --work-dir (directory)
```

Comment:
Right, so given that this is a cBio-specific exporter, could you rename the exporter or move it to the cbioportal tool folder?